### PR TITLE
Completed assignment.

### PR DESCRIPTION
**Pull request by coffeykieran**
Kieran Coffey. 

First, I forked the master repository, and copied it into RStudio on my local computer. Initially I found the syntax for Markdown shown on the cheat sheet challenging to use. But after taking a closer look and playing with the program a bit more I figured it out and wrote up a brief biography. I then pushed my changes in the README.md to Github before creating this pull request. 

Overall I would say this assignment was fairly easy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Kieran Coffey
+I am a MAPSS student studying European History - specifically German cultural and political history of the nineteenth and twentieth centuries. At the moment I am studying the relationship between conceptions of regional, national, and religious identity as they relate to the German concept of *Heimat*, or homeland. My other academic interests include:
+
+    - nationalism and national identity
```
Comment (reed-diana) [inline README.md:6]: nice use of a list here! it's cool that the font and the background changed. Not sure how you did that.

File: README.md [hunk 2]
```diff
@@ -1 +1,18 @@
...
+    - sub-national or regional identity
+    - mass political movements, particularly socialist/communist and far-right movements
+    - borderland regions, particular between Germany and France 
+    and the European Union 
+
+## Outside of Academia
```
Comment (reed-diana) [inline README.md:12]: The section headers work well here, both in terms of format and content.

File: README.md [hunk 3]
```diff
@@ -1 +1,18 @@
...
+I also like to hike, watch soccer, play video games (especially grand stratedgy games! <https://www.paradoxplaza.com>), trying new recipes, and visiting friends in other cities and countries.  
+Originally I'm from Buffalo, New York where I went to highschool and where my family still lives. I went to college at the University of Pittsburgh, and graduated in 2017. I then took language classes at the Goethe Institut in Freiburg im Breisgau, Germany (<https://en.wikipedia.org/wiki/Freiburg_im_Breisgau>) and interned in Stuttgart, Germany at the Heinrich Böll Stiftung (Heinrich Böll Foundation.
```
Comment (reed-diana) [inline README.md:14]: Adding the links here was a good touch, and they worked!

File: README.md [hunk 4]
```diff
@@ -1 +1,18 @@
...
+    - mass political movements, particularly socialist/communist and far-right movements
+    - borderland regions, particular between Germany and France 
+    and the European Union 
+
+## Outside of Academia
+I also like to hike, watch soccer, play video games (especially grand stratedgy games! <https://www.paradoxplaza.com>), trying new recipes, and visiting friends in other cities and countries.  
+Originally I'm from Buffalo, New York where I went to highschool and where my family still lives. I went to college at the University of Pittsburgh, and graduated in 2017. I then took language classes at the Goethe Institut in Freiburg im Breisgau, Germany (<https://en.wikipedia.org/wiki/Freiburg_im_Breisgau>) and interned in Stuttgart, Germany at the Heinrich Böll Stiftung (Heinrich Böll Foundation.  
+I am taking this course for two reasons:
+
+    1. Computer based analytics are being used more in historical research, and I have no previous experience with programming or computer modeling.
```
Comment (reed-diana) [inline README.md:17]: Great to see different kinds of lists in the same document. Not sure if you intended this, but these two lines create a horizontal scroll bar. You could also consider getting the text to wrap.

File: README.md [hunk 5]
```diff
@@ -1 +1,18 @@
...
+I am a MAPSS student studying European History - specifically German cultural and political history of the nineteenth and twentieth centuries. At the moment I am studying the relationship between conceptions of regional, national, and religious identity as they relate to the German concept of *Heimat*, or homeland. My other academic interests include:
+
+    - nationalism and national identity
+    - sub-national or regional identity
+    - mass political movements, particularly socialist/communist and far-right movements
+    - borderland regions, particular between Germany and France 
+    and the European Union 
+
+## Outside of Academia
+I also like to hike, watch soccer, play video games (especially grand stratedgy games! <https://www.paradoxplaza.com>), trying new recipes, and visiting friends in other cities and countries.
```
Comment (reed-diana) [inline README.md:13]: Searching for the required three things to improve on - but adding an image in here might work well. To do that, you need the image file to be in the same folder in both github versions (on the server and on your machine.)

Comment (reed-diana) [inline README.md:13]: This is super minor and you might not even want to do it, but if you are interested there are ways to format your body paragraphs to add space / indent.

File: README.md (hunk 1 above)
Comment (bensoltoff) [reply README.md:6]: It's actually not a correctly processed Markdown list. Don't use indents when initializing a list. Compare this

```
    - nationalism and national identity
    - sub-national or regional identity
    - mass political movements, particularly socialist/communist and far-right movements
    - borderland regions, particular between Germany and France and the European Union 
```

    - nationalism and national identity
    - sub-national or regional identity
    - mass political movements, particularly socialist/communist and far-right movements
    - borderland regions, particular between Germany and France and the European Union 

With this:

```
- nationalism and national identity
- sub-national or regional identity
- mass political movements, particularly socialist/communist and far-right movements
- borderland regions, particular between Germany and France and the European Union 
```

- nationalism and national identity
- sub-national or regional identity
- mass political movements, particularly socialist/communist and far-right movements
- borderland regions, particular between Germany and France and the European Union

File: README.md (hunk 3 above)
Comment (bensoltoff) [inline README.md:14]: When embedding links, it's better to use a text link. That is, don't just paste the link but put it in text to be clicked on (as you would see on most websites)

```
[Goethe Institut in Freiburg im Breisgau, Germany](https://en.wikipedia.org/wiki/Freiburg_im_Breisgau)
```

[Goethe Institut in Freiburg im Breisgau, Germany](https://en.wikipedia.org/wiki/Freiburg_im_Breisgau)

File: README.md (hunk 4 above)
Comment (bensoltoff) [inline README.md:17]: When the list is properly formatted, you don't have to individually number each item. Markdown will do that automatically.

```
1. Computer based analytics are being used more in historical research, and I have no previous experience with programming or computer modeling.
1. I have been told that having some experience with programming languages can improve your chances of getting a job in the social sciences outside of academia. 
```

1. Computer based analytics are being used more in historical research, and I have no previous experience with programming or computer modeling.
1. I have been told that having some experience with programming languages can improve your chances of getting a job in the social sciences outside of academia.

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |           |      x        |            |
| **Presentation**            |           |              |    x        |
| **Achievement, creativity** |           |      x        |            |
| **Reproducibility**         |           |     x         |            |

Good first attempt using Markdown, and the homework was submitted properly!

**Pull request closed without merging**